#goal definition
int16 marker_id
int16 timeout
string sub_topic_name
int8 required_pose_num

---
#result definition

# Note: The expected priority order of the error should match the message order
uint16 UNKNOWN=720
uint16 TIMEOUT=721
uint16 WAITSER_ERROR=722
uint16 INVALID_INPUT=723
uint16 NONE=0

uint16 error_code_id
geometry_msgs/PoseStamped pose

---
#feedback definition
